api/api.py

```diff
@@ -16,195 +22,158 @@ def json_serial(obj):
 
 
 #ESTABLISHING CONNECTION WITH THE POSTGRES DATABASE
-cur= Session.connect_to_psql()
+ss=Session()
+session=ss.connect_to_psql()
+
+
+Blocks=ss.get_table_object('blocks')
+Transactions=ss.get_table_object('transactions')
+Receipts=ss.get_table_object('receipts')
+Uncles=ss.get_table_object('uncles')
+
+
+def json_serial(obj):
```

Comment:
Can we push this method in a utils.py file

Reply:
The json_serial()? Yeah sure, I shall do that in the next commit. Will make the code cleaner.